src/main/java/pl/wojciechkostecki/revel/model/dto/UserDTO.java

```diff
@@ -0,0 +1,23 @@
+package pl.wojciechkostecki.revel.model.dto;
+
+import lombok.Data;
+import pl.wojciechkostecki.revel.model.UserRole;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+import java.util.HashSet;
+import java.util.Set;
+
+@Data
+public class UserDTO {
+
```

Comment:
Worth adding id to this dto object?

Reply:
There was no such need at the moment, but I will add if it should be useful.